test/images/crd-conversion-webhook/converter/framework.go

```diff
@@ -66,6 +70,7 @@ func statusSucceed() metav1.Status {
 // failures will be reported as Reason in the conversion response.
 func doConversion(convertRequest *v1beta1.ConversionRequest, convert convertFunc) *v1beta1.ConversionResponse {
 	var convertedObjects []runtime.RawExtension
+	conversionIterationID++
```

Comment:
this is clever, but not threadsafe (conversion can be called concurrently), and the returned value will get persisted. 

verifying a list gets sent to a conversion webhook seems like a great integration test where we can run the webhook in process and inspect conversions sent to it (possibly something like https://gist.github.com/liggitt/7e7277f73ba4008535033b974b1249fe)

`go test ./vendor/k8s.io/apiextensions-apiserver/test/integration -run TestWebhookConversion -v` is really nice to be able to run as well

Reply:
also, our e2e tests are perpetually close to their max length limit (xref https://github.com/kubernetes/kubernetes/issues/74375), so easing pressure there seems good

Reply:
removed e2e test as discussed and will send a follow up for test, but we know it works.